### PR TITLE
Homework 4 suffix 1

### PR DESCRIPTION
**Pull request by KyuuzoT**
Done

### CODE REVIEW

File: HW_04-1/Task1-1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    return value;
+                }
+                else
+                {
+                    Console.WriteLine($"{input} is not a valid number!");
+                }
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HW_04-1/Task1-1/Program.cs:48]: done

File: HW_04-1/Task2-1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    return value;
+                }
+                else
+                {
+                    Console.WriteLine($"{input} is not a valid number!");
+                }
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HW_04-1/Task2-1/Program.cs:61]: done

File: HW_04-1/Task2-2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Threading.Tasks;
+
+namespace Task2_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        }
+    }
```
Comment (VadzimPapko) [inline HW_04-1/Task2-2/Program.cs:14]: ???

File: HW_04-1/Task3-1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                    return value;
+                }
+                else
+                {
+                    Console.WriteLine($"{input} is not a valid number!");
+                }
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HW_04-1/Task3-1/Program.cs:75]: done

File: HW_04-1/Task4-1/Arithmetic.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            return first * second;
+        }
+
+        public static double Division(double first, double second)
+        {
+            return first / second;
+        }
+    }
+}
+
```
Comment (VadzimPapko) [inline HW_04-1/Task4-1/Arithmetic.cs:26]: done

File: HW_04-1/Task4-1/Program.cs [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+                    return value;
+                }
+                else
+                {
+                    Console.WriteLine($"{input} is not a valid number!");
+                }
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HW_04-1/Task4-1/Program.cs:130]: done

File: HW_04-1/Task2-2/Program.cs (hunk 3 above)
Comment (KyuuzoT) [reply HW_04-1/Task2-2/Program.cs:14]: Accidentally added... Task2-2 hasn't meant to exist.

**Pull request merged**